CHANGES.md

```diff
@@ -10,12 +10,15 @@ Release Notes.
 * Make HttpClient 3.x, 4.x, and HttpAsyncClient 3.x plugins to support collecting HTTP parameters.
 * Make the Feign plugin to support Java 14
 * Make the okhttp3 plugin to support Java 14
+* Support propagate the sending timestamp in MQ plugins to calculate the transfer latency in the async MQ scenarios.
+* Support auto-tag with the fixed values propagated in the correlation context.   
 
 #### OAP-Backend
 * Add the `@SuperDataset` annotation for BrowserErrorLog.
 * Support keeping collecting the slowly segments in the sampling mechanism.
 * Support choose files to active the meter analyzer.
 * Improve Kubernetes service registry for ALS analysis.
+* Add `contain` and `not contain` OPS in OAL.
```

Comment:
Please move `add` to follow the previous `add`. New feature comes first in the change log.